### PR TITLE
Project "NovelTest" specific changes

### PR DESCRIPTION
**Pull request by Crystalwarrior**
See commit descriptions for full changelogs, sry for being lazy

### CODE REVIEW

File: editor/editor.gd [hunk 1]
```diff
@@ -117,7 +117,7 @@ func move_command(command:Command, to_position:int) -> void:
 	
 	var old_position:int = _current_timeline.get_command_idx(command)
 	var action_name:String = "Move command '%s'" % [command.get_command_name()]
-	
+	timeline_displayer.reselect_index = to_position
```
Comment (AnidemDex) [inline editor/editor.gd:120]: This is a good addition that I need to remember to keep

File: editor/timeline_displayer.gd [hunk 2]
```diff
@@ -95,6 +99,7 @@ func _build_item(item:TreeItem, command:Command) -> void:
 	item.set_icon(0, command_icon)
 	
 	item.set_text(1, command_hint)
+	item.set_autowrap_mode(1, TextServer.AUTOWRAP_WORD_SMART)
```
Comment (AnidemDex) [inline editor/timeline_displayer.gd:102]: This is other feature that I need to keep to solve #21

File: icons/false.svg
Comment (AnidemDex) [inline icons/false.svg:1]: I need this icon

File: icons/true.svg
Comment (AnidemDex) [inline icons/true.svg:1]: This other icon too

**Comment by AnidemDex** [comment]
Can't merge, please rebase and solve the conflicts on your branch before. I don't think we need all the new commands, and other conflicting commands are ok on main

File: editor/editor.gd [hunk 3]
```diff
@@ -269,8 +269,15 @@ func _get_file_dialog() -> ConfirmationDialog:
 
 
 func _command_button_list_pressed(command_script:Script) -> void:
+	var command_idx:int = -1
+	var tree_item:TreeItem = timeline_displayer.get_selected()
+	if tree_item:
+		var selected:Command = timeline_displayer.get_selected().get_metadata(0)
+		if selected:
+			command_idx = _current_timeline.get_command_idx(selected) + 1
```
Comment (AnidemDex) [inline editor/editor.gd:277]: TimelineDisplayer now uses CommandCollection, which uses `get_command_index` and `get_command_absolute_position`

File: icons/true.svg
Comment (AnidemDex) [inline icons/true.svg:1]: Later we'll need to be sure that these colors are the same that editor uses. It makes editor changes the color automatically :thinking:

**Review by AnidemDex — CHANGES_REQUESTED** [review]

File: icons/true.svg
Comment (Crystalwarrior) [reply icons/true.svg:1]: I think I did try to make sure of that but we'll have to test

File: editor/editor.gd [hunk 4]
```diff
@@ -269,8 +269,15 @@ func _get_file_dialog() -> ConfirmationDialog:
 
 
 func _command_button_list_pressed(command_script:Script) -> void:
+	var command_idx:int = -1
+	var tree_item:TreeItem = timeline_displayer.get_selected()
+	if tree_item:
+		var selected:Command = timeline_displayer.get_selected().get_metadata(0)
+		if selected:
+			command_idx = _current_timeline.get_command_absolute_position(selected) + 1
```
Comment (AnidemDex) [inline editor/editor.gd:277]: Well, if works, it works

**Review by AnidemDex — APPROVED** [review]

**Comment by AnidemDex** [comment]
Thank you

**Pull request merged**